cli/cli-client/src/main/java/org/infinispan/cli/shell/ShellImpl.java

```diff
@@ -125,18 +126,20 @@ private void batchRun() throws IOException {
    private void interactiveRun() throws IOException {
       config = new ConfigImpl(SystemUtils.getAppConfigFolder("InfinispanShell"));
       config.load();
-      Settings settings = Settings.getInstance();
-      settings.setAliasEnabled(false);
-      console = new Console();
+      SettingsBuilder settings = new SettingsBuilder();
+      settings.enableAlias(false).outputStream(System.out).outputStreamError(System.err).inputStream(System.in);
+      console = new Console(settings.create());
       context.setOutputAdapter(new ConsoleIOAdapter(console));
       console.addCompletion(new Completer(context));
+      console.setConsoleCallback(new CLIConsoleCallback());
```

Comment:
I did another look and I think and the callback will not execute the commands. that is done in the lines below. am I right?